src/main/java/selenium/Driver.java

```diff
@@ -1,18 +1,29 @@
 package selenium;
 
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.edge.EdgeOptions;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
 
 public final class Driver {
     private static Driver instance;
 
-    private WebDriver driver;
+    private final Map<Long, WebDriver> drivers = new HashMap<>();
```

Comment:
if you want to run tests in parallel you need to use thread-safe collections, easiest way in this case to use ThreadLocal here instead of HashMap

Reply:
ThreadLocal is used now